src/maplibre_helpers.ts

```diff
@@ -99,11 +112,13 @@ export function overwritePolygonLayer(
     id: string;
     source: string;
     filter?: FilterSpecification;
+    // For vector (non GeoJSON) sources
+    sourceLayer?: string;
```

Comment:
Vector tile sources themselves can have multiple layers packed in. Even when there's just one, you have to specify it. For the pmtiles we built, they match the filename -- schools.pmtiles -> schools